Ad Management API 1.0 FINAL.md

```diff
@@ -575,6 +609,7 @@ GET `https://api.superads.com/management/v1/bidder/496/ads?auditStart=1528282813
 {
   "count": 100,
   "more": 1,
+  "nextPage": "https://api.superads.com/management/v1/bidder/496/ads?auditStart=1528306991000_557398",
```

Comment:
Just need to update the format on this one to reflect the use of `timestampId`

Reply:
fixed